hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -643,6 +643,14 @@ Consult the javadocs for `org.apache.hadoop.fs.azurebfs.constants.ConfigurationK
 `org.apache.hadoop.fs.azurebfs.AbfsConfiguration` for the full list
 of configuration options and their default values.
 
+### <a href="outputstreamflushconfigoptions"></a> Data Flush Options
+Config fs.azure.disable.outputstream.flush provides an option to render
```

Comment:
can you use backticks to make this config options code formatted. I'm less worried about the others, though really they should all be done too